### PR TITLE
Add icon

### PR DESCRIPTION
**Pull request by stan-dot**

### CODE REVIEW

**Comment by stan-dot** [comment]
DLS logo link 
<https://github.com/DiamondLightSource/python-copier-template/blob/main/docs/images/dls-logo.svg>

File: backstage/packages/app/public/new-favicon.svg
Comment (garryod) [inline backstage/packages/app/public/new-favicon.svg:1]: What does this file do?

File: backstage/packages/app/public/old-favicon.ico
Comment (garryod) [inline backstage/packages/app/public/old-favicon.ico:1]: What does this file do?

**Review by garryod — COMMENTED** [review]
Not sure why additional files have been changed, any chance you could clarify their purpose?

File: backstage/packages/app/public/new-favicon.svg
Comment (stan-dot) [reply backstage/packages/app/public/new-favicon.svg:1]: it's kept for reference before it was converted into `.ico`. Potentially could be renamed and/or moved elsewhere. Still I think it should be kept somewhere in the repo, as it's 'source code' for the 'compiled' `.ico`.

File: backstage/packages/app/public/old-favicon.ico
Comment (stan-dot) [reply backstage/packages/app/public/old-favicon.ico:1]: deleted

**Comment by stan-dot** [comment]
todo: need to change this file https://backstage.io/docs/getting-started/app-custom-theme/#custom-logo

**Comment by stan-dot** [comment]
inksapce issue https://github.com/facebook/docusaurus/issues/3689

**Comment by stan-dot** [comment]
this is excessive effort just for the icon. it's not plug and play and the backstage one is alright

**Pull request closed without merging**